benchmarks/lammps-nvidia/get_performance.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
This doesn't seem to print a score, is 1/wall time sufficient?

Reply:
it writes a yaml file with different performance metrics. 1/wall time is sufficient, but there is also a timesteps/s that is a more standard metric that is to be maximized